AmazonAdvertisingApi/RequestPrefixes.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace AmazonAdvertisingApi;
+
+require_once "Versions.php";
+
+
+class RequestPrefixes
+{
+	private $apiVersion = null;
+
+	public function __construct() 
+	{
+		$versions = new Versions();
+		$this->apiVersion = $versions->versionStrings["apiVersion"];
+	}
+
+    public $requestPrefixStrings = array(
+		"display" => "sd/",
+		"products" => "{$this->apiVersion}/sp/",
+		"brands"   => "sb/",
+		"profile" => "{$this->apiVersion}/",
+		"report" => "{$this->apiVersion}/"
+    );
+}
```

Comment:
My compiler says this is invalid. Declaring a constant using runtime values ($this->apiVersion)

I think either of my two style suggestions from before fix it. Either this being a function or the api version being constants.